designer/apps/pages/models.py

```diff
@@ -51,11 +54,72 @@ class ProgramPage(Page):
     content_panels = Page.content_panels + [
         FieldPanel('uuid', classname="full"),
         FieldPanel('idp_slug'),
+        InlinePanel('program_homepage', label="Program Homepage", max_num=1),
         InlinePanel('program_documents', label="Program Documents", max_num=1),
         InlinePanel('branding', label="Program Page Branding", max_num=1),
     ]
 
 
+class ProgramHomepage(models.Model):
+    """
+    Section on the Program Page linking back to the Program's Homepage and describing what learners can expect to find
+    there.
+    For example the program "Masters in Potions Ingredients Science" this section may include a link to the
+    "hogwarts.edu" homepage and a description of what learners can expect to find at "hogwarts.edu".  For example,
+    "Chat with a professor", "Add or drop a course", "Contact St. Mungo's Poison Control Hotline".
+
+    .. no_pii:
+    """
+    display = models.BooleanField(
+        blank=False,
+        null=False,
+        default=True,
+        verbose_name="Display This Section",
+    )
+    header = models.CharField(
+        max_length=128,
+        verbose_name='Header',
+        blank=False,
+        null=False,
+        default='Manage Your Degree'
+    )
+    description = RichTextField(
+        max_length=512,
+        verbose_name='description',
+        blank=False,
+        null=False,
+        features=('bold', 'italic', 'ol', 'ul'),
+        default="<p>Go to your program's portal to:</p><ul><li>Add or drop courses</li><li>Finance Department</li><li>Contact an advisor</li><li>Get your grade</li><li>Program wide discussions</li><li>and more</li></ul>"  # pylint: disable=line-too-long
```

Comment:
Nit: Can we avoid using `# pylint: disable=line-too-long` here?

Reply:
I was actually not sure the best approach to doing this we can use python's multiline string but it adds a bunch of ```\n``` and whitespace into the string.  I'm not sure if there is a nice way in python to create an HTML string

Reply:
I can't think of any super clean way. You could just a constant to your class like
```
DEFAULT_DESCRIPTION = re.sub('>\s+<', '><', 
"""
<p>Go to your program's portal to:</p>
<ul>
    <li>Add or drop courses</li>
    <li>Finance Department</li>
    <li>Contact an advisor</li>
    <li>Get your grade</li>
    <li>Program wide discussions</li>
    <li>and more</li>
</ul>
""").lstrip().rstrip()
``` 
and just set `default=DEFAULT_DESCRIPTION` ?

Reply:
oh yeah that is a good way to do that - thanks!